icssc-projects/AntAlmanac
# Add 'Clear Schedule' buttons

### Summary
A 'Clear Schedule' button was added to the "Added Classes" section.
A trash icon was also added to the left pane bar for clearing the schedule.

Clicking either button will prompt the user to confirm that they want to delete their schedule. If the user confirms, the current schedule will be cleared.

Screenshot of the two new buttons with the mouse hovering over the trash icon:
![image](https://user-images.githubusercontent.com/29494270/100498598-37cba080-3118-11eb-9bdf-3b06f4619993.png)

Screenshot of the confirmation prompt:
![image](https://user-images.githubusercontent.com/29494270/100498752-5bdbb180-3119-11eb-91f9-11be51661b92.png)

### Test Plan
For both buttons, the following was tested:
- Add a new course and a new custom event to the schedule
- Click the button to clear the schedule
- Verify that the confirmation prompt appears
- Test both the confirm and deny actions:
    - Confirm that you would like to clear the schedule
        - Verify that the course and custom event are deleted
        - Verify that other schedules were not affected
    - Decline the prompt
        - Verify that the schedule is _not_ cleared


### Issues
Closes #32
